modules/network-observability-netobserv-dashboard-custom-alerts.adoc

```diff
@@ -0,0 +1,37 @@
+//
+// network_observability/configuring-operator.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-netobserv-dashboard-custom-alerts_{context}"]
+= Create custom NetObserv dashboard alerts
+You can create custom alerts for the *Netobserv* dashboard. For more information, see _Creating alerting rules for user-defined projects_ in the _Monitoring_ documentation. 
+
+An example of an alerting rule configuration YAML file is as follows:
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: alert-external-traffic
+  namespace: netobserv
+spec:
+  groups:
+  - name: AlertExternalTraffic
+    rules:
+    - alert: MediumTraffic
+      annotations:
+      description: Amount of traffic from/to cluster external workloads is medium.
+      expr: sum(rate(netobserv_workload_ingress_bytes_total{DstK8S_OwnerType=""}[1m])) > 150
+      for: 1m
+      labels:
+        app: netobserv
+        severity: info
+    - alert: HighTraffic
+      annotations:
+      description: Amount of traffic from/to cluster external workloads is high.
+      expr: sum(rate(netobserv_workload_ingress_bytes_total{DstK8S_OwnerType=""}[1m])) > 100000
+      for: 1m
+      labels:
+        app: netobserv
+        severity: critical
```

Comment:
I'll need to find a better example, as the values here are not credible in  a real-world scenario

Reply:
@jotak OK that would be awesome. Even though I link to the Monitoring doc, which has an example, I think if we could have a good example it would be useful. The Monitoring example is also pretty generic. I'm not sure if that's because real-world examples may be hard to exemplify...